cfme/infrastructure/virtual_machines.py

```diff
@@ -406,8 +406,11 @@ def create(self):
             if self.name is not None:
                 snapshot_dict['name'] = self.name
 
-            if self.vm.provider.mgmt.is_vm_running(self.vm.name):
```

Comment:
**REQUIRED**
put 
```
if self.vm.provider.mgmt.is_vm_running(self.vm.name):
   test_memory = True
```
before instead

Reply:
So test_memory is meant to be a flag, that when set, we don't check if the vm is running. This flag is set by a specific test that is designed to try and set the memory even if the vm is powered off. This throws an exception.

So when `test_memory == True` we don't check if `self.vm.provider.mgmt.is_vm_running(self.vm.name)`.